### ISSUE TITLE
Cover

### ISSUE THREAD

**Issue by nichtich** [pro4bib/handbuch-it-in-bibliotheken#15]
Das [Papierfischchen](https://de.wikipedia.org/wiki/Papierfischchen) als cover-Tier wäre zu makaber, oder?

**Comment by lambo** [comment]
Nee, gib Mal der Grafikerin ne Chance, die jetzt von der TIB dafür beauftragt wurde... Ich schätze dass sie bis nächste Woche oder so ein oder zwei hübsche Vorschläge machen wird :)

**Comment by nichtich** [comment]
Super! Wäre gut wenn auch ein passendes Favicon und ein Open Graph Image herauskommt. In der Navbar oben links kann ebenfalls ein Logo angezeigt werden. Open Graph Image und Logo sind eher Querformat.

**Comment by nichtich** [comment]
Noch nicht erledigt aber Teil von #124
